src/layouts/default.vue

```diff
@@ -38,8 +39,5 @@ export default Vue.extend({
     this.$inkline.config.variant = theme.getUserTheme();
     this.$store.dispatch("zk-provider/requestProvider");
   },
-  beforeDestroy() {
```

Comment:
Why u're dropping it?

Reply:
Because it didn't even work, this dispatch doesn't exist. Even if it would have worked it still doesn't have much sense :)